rdflib/plugins/serializers/trig.py

```diff
@@ -40,22 +41,6 @@ def preprocess(self):
 
             self._contexts[context]=(self.orderSubjects(), self._subjects, self._references)
 
-
-    def preprocessTriple(self, triple):
-        s, p, o = triple
-        self._references[o]+=1
-        self._subjects[s] = True
-        for i, node in enumerate(triple):
-            if node in self.keywords:
-                continue
-            # Don't use generated prefixes for subjects and objects
-            self.getQName(node, gen_prefix=(i == VERB))
-            if isinstance(node, Literal) and node.datatype:
-                self.getQName(node.datatype, gen_prefix=_GEN_QNAME_FOR_DT)
-        p = triple[1]
-        if isinstance(p, BNode):
-            self._references[p]+=1
-
```

Comment:
did you remove this on purpose? seems it is called from `preprocess` and the one in `TurtleSerializer` looks quite different to me